frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunDetails.tsx

```diff
@@ -1,7 +1,9 @@
 import * as React from 'react';
-import { SectionHeading, ResourceSummary } from '@console/internal/components/utils';
+import { SectionHeading, ResourceSummary, ResourceLink } from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
 import PipelineRunVisualization from './PipelineRunVisualization';
 import { PipelineRun } from '../../../utils/pipeline-augment';
+import { PipelineModel } from '../../../models/index';
```

Comment:
Ah, missed this in my previous review. You'll almost never need to specify `/index` on any JavaScript (or TypeScript) import because importing from a JS module will cause it to default to the `index` file.